src/src_main/sadb_routine_inmemory.template.c

```diff
@@ -19,15 +19,15 @@ static int32_t sadb_config(void);
 static int32_t sadb_init(void);
 static int32_t sadb_close(void);
 // Security Association Interaction Functions
-static int32_t sadb_get_sa_from_spi(uint16_t, SecurityAssociation_t **);
-static int32_t sadb_get_operational_sa_from_gvcid(uint8_t, uint16_t, uint16_t, uint8_t, SecurityAssociation_t **);
-static int32_t sadb_save_sa(SecurityAssociation_t *sa);
+static int32_t sadb_get_sa_from_spi(uint16_t, SecurityAssociation_t* *);
+static int32_t sadb_get_operational_sa_from_gvcid(uint8_t, uint16_t, uint16_t, uint8_t, SecurityAssociation_t* *);
```

Comment:
SecurityAssociation_t** is better... The * * with the space concerns me -- probably works?

(the same * * thing happens in a number of places